github_connector/models/abstract_github_model.py

```diff
@@ -143,8 +143,10 @@ def get_from_id_or_create(self, data, extra_data=None):
 
         # We try to see if object exist by name (instead of id)
         if self._github_login_field and self._github_login_field in data:
-            existing_object = self.search([
-                ('github_login', '=', data[self._github_login_field])])
+            existing_object = self.search(
+                [('github_login', '=', data[self._github_login_field])],
+                active_test=False,
```

Comment:
This should be passed by context, isn't it?

Reply:
@sbidoul I agree with @pedrobaeza `active_test=False` should be passed with a `with_context(active_test=False).search()`
AFAIK `active_test`is not a valid parameter of the `search` method https://github.com/odoo/odoo/blob/11.0/odoo/models.py#L1465